### PR TITLE
Added L6b glutamatergic neuron

### LINKED ISSUE #881: [NTR] L6b glutamatergic neuron
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
L6b glutamatergic neuron

**Synonyms**
L6b

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
Glutamatergic neuron located in cortical layer 6b
PMID:10.1101/2020.10.19.34312

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
glutamatergic neuron

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
http://purl.obolibrary.org/obo/UBERON_0000956 - Cerebral Cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**
xref:(ILX:0770163)
Related to #827

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #881

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27533,6 +27515,16 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 <https://identifiers.org/ensembl:ENSMUSG00000005716>))
 
+# Class: obo:CL_4023038 (L6b glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.34312") Annotation(oboInOwl:hasDbXref "PMID:30382198") obo:IAO_0000115 obo:CL_4023038 "Glutamatergic neuron located in cortical layer 6b. They transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:27520]: Small changes: "A glutamatergic neuron located in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate."

**Review by patrick-lloyd-ray — COMMENTED** [review]
Small changes.

**Review by dosumis — CHANGES_REQUESTED** [review]
Agree with Patrick's suggested changed to def.

**Review by shawntanzk — COMMENTED** [review]
> Agree with Patrick's suggested changed to def.

Have already changed it accordingly

**Review by dosumis — APPROVED** [review]

**Pull request merged**